### PR TITLE
[contibutors] add yaml entry.

### PR DESCRIPTION
**Pull request by surahman**
@qianzhangxa I have added my mapping information to the contributor file, thank you!

### CODE REVIEW

File: docs/contributors.yaml [hunk 1]
```diff
@@ -963,3 +963,9 @@
     - bluezhudong@gmail.com
   jira_user: dzhu
   reviewboard_user: dzhu
+
+- name: Saad Ur Rahman
+  emails:
+    - saad.ur.rahman@gmail.com
+  jira_user: surahman
+  reviewboard_user: saad.ur.rahman
```
Comment (qianzhangxa) [inline docs/contributors.yaml:971]: This file is in alphabetical order, so I do not think your info should be added in the bottom.

Comment (surahman) [reply docs/contributors.yaml]: @qianzhangxa sorry about that, I very quickly tail appended the data 🤦🏽. I have fixed it.

Comment (qianzhangxa) [reply docs/contributors.yaml]: @surahman No problem, I have merged it.

Comment (surahman) [reply docs/contributors.yaml]: @qianzhangxa thanks :)

**Pull request closed without merging**